drivers/hmis/app/graphql/types/ac_hmis/mci_clearance_match.rb

```diff
@@ -0,0 +1,42 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class AcHmis::MciClearanceMatch < Types::BaseObject
+    include Types::HmisSchema::HasGender
+
+    field :id, ID, null: false
+    field :mci_id, String, null: false
+    field :score, Integer, null: false
+    field :existing_client_id, ID, 'ID of existing Client that has the same MCI ID', null: true
```

Comment:
Is there ever a situation where we'd have multiple clients with the same MCI ID?  Does it just choose the first?

Reply:
If there are multiple, it chooses the one with the earlier creation that. That was specified by the client.

As noted [in this TODO](https://github.com/greenriver/hmis-warehouse/blob/a542f4bc5d960884a5c4e2fc1fed27cd08288c2d/drivers/hmis_external_apis/app/models/hmis_external_apis/mci.rb#L63-L64) we will likely also update the logic so that it looks for a match based on the MCI Unique ID, too.